src/contracts/Persistence/Content/Handler.php

```diff
@@ -283,7 +283,7 @@ public function removeRelation($relationId, $type, ?int $destinationContentId =
     /**
      * Loads relations from $sourceContentId. Optionally, loads only those with $type and $sourceContentVersionNo.
      *
-     * @deprecated 4.5.7 The "ContentService::loadRelations()" method is deprecated, will be removed in 5.0.
+     * @deprecated 4.5.7 The {@see \Ibexa\Contracts\Core\Repository\ContentService::loadRelations()} method is deprecated, will be removed in 5.0.
```

Comment:
The same remark as above.

Reply:
@alongosz You're right!

```suggestion
     * @deprecated 4.5.7 Method is deprecated since 4.5.7, and will be removed in 5.0. Use {@see \Ibexa\Contracts\Core\Repository\ContentService::loadRelationList()} instead.
```